lib/player.ex

```diff
@@ -0,0 +1,27 @@
+defmodule Player do
+  defstruct [:marker]
+
+  def get_move(marker, board) do
+    get_valid_move(marker, board)
+  end
+
+  def get_valid_move(marker, board) do
+    get_cell_location(marker)
+    |> Validator.is_valid_move(board)
+    |> get_valid_location(marker, board)
+  end
+
+  def get_valid_location(is_valid, _marker, _board) when elem(is_valid, 0) == true do
+    elem(is_valid, 1)
+  end
+
+  def get_valid_location(is_valid, marker, board) when elem(is_valid, 0) == false do
+    :cell_filled |> Messages.get() |> Console.display()
+    get_valid_move(marker, board)
+  end
+
+  def get_cell_location(marker) do
+    Messages.get(:get_move, marker)
+    |> Console.get_input()
+  end
+end
```

Comment:
I'm still thrown off by the use of `valid` here in Player since there's a `Validator` class.

Reply:
I agree with @scottyplunkett something is not quite right here